include/revng/Pipeline/Container.h

```diff
@@ -78,6 +78,9 @@ class ContainerBase {
   /// The implementation must esure that the content of this file will be
   /// loaded from the provided path.
   virtual llvm::Error loadFromDisk(llvm::StringRef Path);
+
+  /// Checks that the content of the this container is well formed.
```

Comment:
`s/well formed/valid/`
I'm trying to push a terminology similar to XML, where "well formed" means that it's parsable but might fail to respect a schema. If it validates a schema it's "valid". Here we're more in the latter situation.